### PR TITLE
Add installer popups

### LINKED ISSUE #206: Add popup message for installers
## Instructions

When you install the extension for the first time it says you need to install CLI and you can click install which is great, but then the popup disappears and you have no indication whether it was installed or if that’s still running. Any kind of message saying installing please wait and then finished installing or something like that would be great if possible.

### PR DESCRIPTION
**Pull request by DylanTinianov**
Closes #206 

## Description
Added more thorough popups which guide the user through dependency installations
______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/cadence/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: extension/src/dependency-installer/installer.ts [hunk 1]
```diff
@@ -26,13 +26,16 @@ export abstract class Installer {
       return
     }
 
-    DEBUG_LOG('Running ' + this.#installerName + ' installer...')
+    void window.showInformationMessage('Running ' + this.#installerName + ' installer...')
```
Comment (sideninja) [inline extension/src/dependency-installer/installer.ts:29]: Maybe add: "Running CLI installer, please wait..."

Comment (sideninja) [inline extension/src/dependency-installer/installer.ts]: Do you think we could use progress indication "withProgress" -> https://code.visualstudio.com/api/references/vscode-api

Comment (DylanTinianov) [reply extension/src/dependency-installer/installer.ts]: Done

Comment (DylanTinianov) [reply extension/src/dependency-installer/installer.ts]: The installation is very quick, I don't think it's necessary to add that complexity.

**Review by sideninja — APPROVED** [review]

**Pull request merged**